### DISCUSSION TITLE
Add support for `bg-[paint(...)]`

### DISCUSSION THREAD

**Discussion by devlzcode** [tailwindlabs/tailwindcss#12402] [Ideas]
Expected behavior should be identical to `bg-[url(...)]` which resolves to `background-image: url(...)`. Actual behavior is `bg-[paint(...)]` resolves to `background-color: paint(...)` which is not valid. Workaround is to either inline the style or `[background-image:paint(...)]`.

**Comment by wongjn** [comment]
To note, another work-around is to type-cast explicitly as an image like `bg-[image:paint(…)]`.
